### ISSUE TITLE
How can I use django snippets?

### ISSUE THREAD

**Issue by askfiy** [rafamadriz/friendly-snippets#138]
I use vsnip:

```
vim.g.vsnip_filetypes = {
    ["javascript"] = {"typescript"},
    ["typescript"] = {"javascript"},
    ["vue"] = {"javascript", "typescript", "html"},
    ["html"] = {},
    ["python"] = {}
}
```
I don't know how to set it up, I tried:

```
["python"] = {'frameworks.django.imports'}

["python"] = {'django.imports'}

["python"] = {'imports'}
```

But unfortunately, none of them work.

**Comment by askfiy** [comment]
Can you tell me what to do?

**Comment by rafamadriz** [comment]
Hey @askfiy, sorry for the delay.

In vimscript it should be like this: 

```vim
let g:vsnip_filetypes.python = ['django']
```

And lua I think it should be like this:

```lua
vim.g.vsnip_filetypes.python = {'django'}
```

**Comment by askfiy** [comment]
Sorry, I use vsnip, and the `vsnip_filetypes` option is set:

```
vim.g.vsnip_filetypes = vim.fn.stdpath("config") .. "/snippets"
```

I followed the information you gave and set up:

```
vim.cmd("let g:vsnip_filetypes.python = ['django']")
```

But got the following error:

```
Error detected while processing TextChangedI Autocommands for "*":
E5108: Error executing lua Vim(for):E715: Dictionary required
stack traceback:
        [C]: in function 'vsnip#get_complete_items'
        ...im/site/pack/packer/opt/cmp-vsnip/lua/cmp_vsnip/init.lua:19: in function 'complete'
        ...re/nvim/site/pack/packer/opt/nvim-cmp/lua/cmp/source.lua:296: in function 'complete'
        ...hare/nvim/site/pack/packer/opt/nvim-cmp/lua/cmp/core.lua:293: in function 'complete'
        ...hare/nvim/site/pack/packer/opt/nvim-cmp/lua/cmp/core.lua:168: in function 'callback'
        ...hare/nvim/site/pack/packer/opt/nvim-cmp/lua/cmp/core.lua:218: in function 'autoindent'
        ...hare/nvim/site/pack/packer/opt/nvim-cmp/lua/cmp/core.lua:160: in function 'on_change'
        ...hare/nvim/site/pack/packer/opt/nvim-cmp/lua/cmp/init.lua:328: in function 'callback'
        .../site/pack/packer/opt/nvim-cmp/lua/cmp/utils/autocmd.lua:31: in function 'emit'
        [string ":lua"]:1: in main chunk
```

How can I solve this problem?

**Comment by rafamadriz** [comment]
@askfiy please provide a reproducible example.

**Comment by askfiy** [comment]
@rafamadriz 

```
set rtp+=~/.local/share/nvim/site/pack/packer/opt/nvim-cmp
set rtp+=~/.local/share/nvim/site/pack/packer/opt/cmp-buffer
set rtp+=~/.local/share/nvim/site/pack/packer/opt/cmp-nvim-lsp
set rtp+=~/.local/share/nvim/site/pack/packer/opt/vim-vsnip
set rtp+=~/.local/share/nvim/site/pack/packer/opt/cmp-vsnip
set rtp+=~/.local/share/nvim/site/pack/packer/opt/nvim-lspconfig
set rtp+=~/.local/share/nvim/site/pack/packer/opt/friendly-snippets

" ~/.local/share/nvim/site/pack/packer/opt/
set completeopt=menuone,noselect

lua <<EOF
-- Setup nvim-cmp.
local cmp = require'cmp'

vim.g.vsnip_filetypes = {
  python = {"django"}
  }


cmp.setup({
snippet = {
  expand = function(args)
  vim.fn["vsnip#anonymous"](args.body)
  end,
  },
mapping = {
  ['<CR>'] = cmp.mapping.confirm({ select = true }),
  ['<tab>'] = cmp.mapping(cmp.mapping.confirm({ select = true }), { 'i', 's', 'c' }),
  },
sources = {
  {name = "vsnip"},
  { name = 'nvim_lsp' },
  { name = 'buffer' },
  },
})
EOF

lua << EOF
local capabilities = require('cmp_nvim_lsp').update_capabilities(vim.lsp.protocol.make_client_capabilities())

require'lspconfig'.pyright.setup{
capabilities = capabilities,
root_dir = function()
return vim.fn.getcwd()
end
}
EOF

runtime! after/plugin/*.lua
```

**Comment by askfiy** [comment]
This problem still exists. Do I have a problem configuring it. It's been over twenty days. I hope someone responds.

**Comment by askfiy** [comment]
This branch has fixed the issue [#146](https://github.com/rafamadriz/friendly-snippets/pull/146)
